test/unit/src/common/actions/t_repositories.js

```diff
@@ -32,20 +32,42 @@ describe('repositories actions', () => {
   });
 
   context('fetchRepositoriesSuccess', () => {
-    let payload = [ { fullName: 'test1' }, { fullName: 'test2' }];
+    let result = {
+      payload: {
+        entities: {
+          repos: {
+            123: { fullName: 'test1' },
+            456: { fullName: 'test2' }
+          }
+        },
+        result: {
+          0: 123,
+          1: 456
+        }
+      },
+      pageLinks: {
```

Comment:
It's a bit weird that pageLinks are outside of payload - it also makes action more dependant on structure of the response.

I think that during refactoring pageLinks should be moved into payload together with rest of normalized response.